src/clj_jargon/init.clj

```diff
@@ -71,17 +91,18 @@
     (let [aof  (.getIRODSAccessObjectFactory cm-proxy)
           acnt (.getAuthenticatedIRODSAccount (.authenticateIRODSAccount aof (account cfg client-user)))]
       (assoc cfg
-        :irodsAccount        acnt
-        :accessObjectFactory aof
-        :collectionAO        (.getCollectionAO aof acnt)
-        :dataObjectAO        (.getDataObjectAO aof acnt)
-        :userAO              (.getUserAO aof acnt)
-        :userGroupAO         (.getUserGroupAO aof acnt)
-        :fileFactory         (.getIRODSFileFactory cm-proxy acnt)
-        :fileSystemAO        (.getIRODSFileSystemAO aof acnt)
-        :lister              (.getCollectionAndDataObjectListAndSearchAO aof acnt)
-        :quotaAO             (.getQuotaAO aof acnt)
-        :executor            (.getIRODSGenQueryExecutor aof acnt)))
+        :irodsAccount         acnt
+        :accessObjectFactory  aof
+        :ticketServiceFactory (TicketServiceFactoryImpl. aof)
```

Comment:
This seemed like a more consistent way to create the ticket service factory, even though it doesn't have a convenience method on the AO factory like the other stuff.